### PR TITLE
GPS Fallback

### PR DESCRIPTION
**Pull request by indragiek**
Fallback for entering GPS coordinates when there is no network and the place picker can't be shown.

screen shot 2015-04-06 at 8 14 37 pm

### CODE REVIEW

File: Team9Project/res/layout/alert_gps.xml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        android:labelFor="@id/et_longitude"
+        android:selectAllOnFocus="true" />
+    
+    <Button
+        android:id="@+id/btn_current_location"
+   		android:layout_width="wrap_content"
+    	android:layout_height="wrap_content"
+    	android:layout_marginTop="10dp"
+    	android:text="@string/current_location_button_title" />
+
```
Comment (mrthegreensky) [inline Team9Project/res/layout/alert_gps.xml:34]: Indentation?

Comment (indragiek) [inline Team9Project/res/layout/alert_gps.xml:34]: Yep, fixed

**Comment by mrthegreensky** [comment]
This production line is out of control! Good to roll out!

**Comment by indragiek** [comment]
:)

**Pull request merged**